### DISCUSSION TITLE
MessageSegment.poke的type_是什么，要怎么填

### DISCUSSION THREAD

**Discussion by lmst2** [nonebot/discussions#295] [问答]
如题，看到文档里说是消息段类型，但是都有什么类型呢，我该怎么填写，一概不知，请知道的人帮我解答一下

**Comment by yanyongyu** [comment]
poke的种类请看gocq或者onebot文档，gocq好像不支持这个消息段

**Comment by zty012** [comment]
看gocq文档
http://skaa.nndx.ml/cqcode/#%E5%90%88%E5%B9%B6%E8%BD%AC%E5%8F%91%E6%B6%88%E6%81%AF%E8%8A%82%E7%82%B9
